yaml/thz504.yaml

```diff
@@ -5,3 +5,17 @@ esphome:
 
 packages:
   BASE: !include { file: wp_base.yaml}
+
+  HEIZGASTEMPERATUR:              !include { file: wp_temperature.yaml, vars: { property: "HEIZGASTEMPERATUR"       , disabled_by_default: "true" }} 
+  VERFLUESSIGERTEMPERATUR:        !include { file: wp_temperature.yaml, vars: { property: "VERFLUESSIGERTEMPERATUR" , disabled_by_default: "true" }} 
+  OELSUMPFTEMPERATUR:             !include { file: wp_temperature.yaml, vars: { property: "OELSUMPFTEMPERATUR"      , disabled_by_default: "true"}} 
```

Comment:
Please disable all new entities by default. We can discuss the base setup separately